full-stack-tests/presentation/src/frontend/providers/TableDataProvider.test.ts

```diff
@@ -167,7 +167,7 @@ describe("TableDataProvider", async () => {
           value: {
             valueFormat: PropertyValueFormat.Array,
             itemsTypeName: "Physical Object",
-            items: (value: PropertyRecord[]) => value.length === 2,
+            items: (value: PropertyRecord[]) => value.length === 2 && value.every((item) => Object.keys((item.value as StructValue).members).length > 0),

```

Comment:
Kind of hard to understand when the check is doing. I suggest adding a function `isStructWithMembers(record: PropertyRecord): boolean` and then using it here `value.every(isStructWithMembers)`.

Reply:
Done